apple-pie/styles/style.css

```diff
@@ -0,0 +1,54 @@
+body {
+  padding: 0;
+  margin: 0;
+  overflow-x: hidden;
+  }
+
+html {
+    font-family: 'Open Sans', sans-serif;
+    font-size: 16px;
+}
+
+ul, ol {
+    list-style: none;
+}
+
+#header {
+  background-image: url(/lab-css-recipes-clone/apple-pie/images/apple-pie.jpg);
```

Comment:
this one is even more trickier, when it comes to picking the correct url because we are already within /styles. 
Best to read this again:
https://stackoverflow.com/questions/2005079/absolute-vs-relative-urls
and think about what that means in this case. 
Right would be "/apple-pie/images/apple-pie.jpg"

Reply:
Thank you.